version/version.proto

```diff
@@ -20,5 +20,5 @@ package gnsi;
 
 import "github.com/openconfig/gnoi/types/types.proto";
 
-option (gnoi.types.gnoi_version) = "1.2.0";
+option (gnoi.types.gnoi_version) = "1.3.0";
```

Comment:
i have no idea what this does ... was there some complaint thingy in the commit / presubmit that said change this? (or someone else that said to change this?)

Reply:
Yup, I don't have any opinion about versioning but given it's an API change niranjan said I should change that in the comments.